libsoftwarecontainer/src/gateway/devicenode/devicenodegateway.cpp

```diff
@@ -51,38 +51,29 @@ bool DeviceNodeGateway::activateGateway()
     }
 
     for (auto &dev : devlist) {
-        log_info() << "Mapping device " << dev.name;
-
-        std::string devicePathInContainerOnHost = buildPath(getContainer()->rootFS(), dev.name);
-        std::string deviceParent = parentPath(devicePathInContainerOnHost);
-
-        // Already existing files can't be converted into directories
-        if (existsInFileSystem(deviceParent) && !isDirectory(deviceParent)) {
```

Comment:
Those lines are removed, because we are no longer support creating and mounting new devices.